test/utils/css.js

```diff
@@ -0,0 +1,7 @@
+// @flow
```

Comment:
for now let's move these variables into our only test file

Reply:
Ok - no problem, from experience - I know that it is always better clubbing all selectors in one file but as we have only one that's fine for now